jest-preset.json

```diff
@@ -8,7 +8,10 @@
   "transform": {
     "^.+\\.(ts|js|html)$": "<rootDir>/node_modules/jest-preset-angular/preprocessor.js"
   },
-  "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|js)$",
+  "testMatch": [
+    "**/__tests__/**/*.(ts|js)?(x)",
+    "**/+(*.)(spec|test).(ts|js)?(x)"
```

Comment:
we should also add `+`s before the `(..|..)` groups, because that's a valid glob. What we use currently at Jest (and I filed a PR to fix it: https://github.com/facebook/jest/pull/5744) uses an extra feature from `micromatch` library.
```
"**/__tests__/**/*.+(ts|js)?(x)",
"**/+(*.)+(spec|test).+(ts|js)?(x)"
```